microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -29,15 +29,15 @@ Websites can use the built-in Internationalization API with a regional code to p
 
 In general, the website's internationalization user experiences are only as good as the regional code that they're able to determine for the current user. 
 
-A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  See [ICU](https://icu.unicode.org/home).
+A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  For more information, see [ICU-TC Home Page](https://icu.unicode.org/home).
 
 To set the user's language preferences, Microsoft Edge gets the default values from the OS when it is first installed. The user can then change the language preferences in `edge://settings/languages` and sort them in priority order.
```

Comment:
pls move the new "Preferred languages" screenshot to near here (it's mentioned in this sentence & table, maybe elsewhere too), and restore the "OS regional" screenshot.  Note that in the UI, "Preferred lang" is at top of UI and "OS region" is at bottom.  I looked at them both at once onscreen in the UI but they are spread too far apart to fit into a single capture at 100% zoom level.  They are not adjacent in the UI else single capture would be perfect.

Reply:
done

Reply:
We're moving into styleist changes, but yes I did this one.